tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -43,6 +43,35 @@
   register: edpm_privatekey
   when: edpm_encoded_privatekey is undefined
 
+# Remove this when https://code.engineering.redhat.com/gerrit/q/topic:stable-compute-uuid
+# is released in 17.x
+- name: Temporary fix to ensure stable compute UUID
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    declare -A computes
+    computes=(
+      ["{{ edpm_node_hostname }}.localdomain"]="{{ edpm_node_ip }}"
+    )
+
+    for name in "${!computes[@]}";
+    do
+      uuid=$(\
+        oc exec -t openstackclient -- \
```

Comment:
I think we need to put IDs fetched from the source cloud, not the destining one. That would ensure adoption fails if uuid changes.

Reply:
At this point of the process the DB is adopted from the the source cloud with all the IDs in it, but now EDPM adoption was done, so no compute services are running in the source compute. So the IDs in the dest DB is the same as the IDs in the source DB.